docs/2.0 EDID to Receiver Caps mapping.md

```diff
@@ -2,38 +2,41 @@
+Extended Display Identification Data (EDID) is a metadata format for sink devices to describe their capabilities to a video source (e.g. graphics card or set-top box). The data format is defined by a standard published by the Video Electronics Standards Association (VESA). This document is targeted against [E-EDID A2][E-EDID] files which consist of EDID 1.4 which is called Base EDID and an optional [CTA-861-G][CTA-861] Extension Block.
+
+[AMWA BCP-004-01][BCP-004-01] describes a methodology for a similar purpose in relationship to [AMWA IS-04][IS-04].
 
 ## Scope
 
-This document is targeted against [E-EDID A2][E-EDID] files which consist of EDID 1.4 which is called Base EDID and an optional [CTA-861-G][CTA-861] Extension Block.
+This document provides Best Current Practice guidelines for how to implement a mapping of EDID fields to Receiver Capabilities in order to have a specific way of representing an EDID sink devices through an associated IS-04 Receiver.
 
 ## Use of Normative Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][RFC-2119].
 
 ## Definitions
 
-_See also the [NMOS Glossary](https://github.com/AMWA-TV/nmos/wiki/Glossary), and definitions within RFCs._
+_See also the [NMOS Glossary](https://specs.amwa.tv/nmos/branches/main/docs/4.0._Glossary.html), and definitions within RFCs._
 
 ### Gateway Device
 
-A device which encapsulates digital video or/and audio (usually, from a physical device) into IP packets or/and does the opposite operation.
+A device which encapsulates a digital video or/and audio signal (usually, from a physical connection) into IP packets. Conversely, the device may perform the reverse operation for IP packets to a digital signal.
+
 ### Sink
 
 A media consuming unit associated with an IS-04 Receiver.
 
 ## Video Receivers
 
-If an IS-04 Video Receiver is associated with a Sink which has EDID, its video capabilities MUST be mapped into Receiver's Receiver Capabilities accordingly to the rules below.
+If an IS-04 Video Receiver is associated with a Sink which has EDID, its video capabilities MUST be mapped into Receiver's Capabilities accordingly to the rules below.
 
-If the Sink acts as a Gateway Device, it's supposed to expose its own EDID instead of EDID of a physical device to take into account any media transformations which the Sink performs.
+If the Sink acts as a Gateway Device, it's supposed to expose its own EDID and therefore Receiver Capabilities and not take into account EDIDs of Sinks connected to the Gateway in a non-networked way. :eyes:
```

Comment:
Hi, @prince-chrismc.
I tried to explain the idea in a different way. Could you take a look, please?
```suggestion
If some processing happens between a Receiver and its Sink, (e.g. the Receiver is able to consume 4K Flow and downscale it for the Sink with only Full HD in its EDID), Receiver Capabilities MUST reflect not taking EDID as is. This can also be said for Receivers associated with multiple Sinks at the same time.
```

Reply:
👍